Gordon360/Controllers/RecIM/ActivitiesController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Services.RecIM;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Gordon360.Controllers.RecIM
+{
+    [Route("api/recim/[controller]")]
+    [AllowAnonymous]
+    public class ActivitiesController : GordonControllerBase
+    {
+        private readonly IActivityService _activityService;
+
+        public ActivitiesController(IActivityService activityService)
+        {
+            _activityService = activityService;
+        }
+
+        ///<summary>Gets a list of all Activities by parameter </summary>
+        ///<param name="active"> Optional active parameter </param>
+        ///<param name="time"> Optional time parameter </param>
+        /// <returns>
+        /// All Existing Activities 
+        /// </returns>
+        [HttpGet]
+        [Route("")]
+        public ActionResult<IEnumerable<ActivityViewModel>> GetActivities([FromQuery] DateTime? time, bool active)
+        {   
+            if (time is null && active)
+            {
+                var activeResults = _activityService.GetActivities();
+                return Ok(activeResults);
+                
+            }
+            var result = _activityService.GetActivitiesByTime(time);
+            return Ok(result);
+        }
+
+        ///<summary>Gets a Activity object by ID number</summary>
+        /// <param name="activityID">League ID Number</param>
```

Comment:
I hate to add more overloaded terms to our code base. Can you remind me why you are using terminology for both `league` and `activity` here?

Reply:
Whoops, that was an oversight. Every instance of league should have been changed but I guess I didn't look too carefully in the documentation. I'll make the adjustment in the next PR.

Reply:
Ah okay. I did see it somewhere else as well but didn't comment for it.

Reply:
I'll look for it and add it as a change on the next PR